lib/imageDisplay.ml

```diff
@@ -122,12 +122,10 @@ let print_portrait conf base p =
     [Rem] : Ne pas utiliser en dehors de ce module.                           *)
```

Comment:
since `print_source` is now in the .mli, can you update this comment and move it to the mli